v4.5/Makefile

```diff
@@ -0,0 +1,322 @@
+PROFILE := v4.5
```

Comment:
This is getting confusing. If you want to add 3 sets of 4.5 how can this be used for all of them when 4.5 minor versions have different APIs?